src/DynamoCore/Models/WorkspaceModel.cs

```diff
@@ -1099,7 +1121,7 @@ private void GenerateCombinedGraph(bool isGroupLayout)
                 foreach (AnnotationModel group in DynamoSelection.Instance.Selection.OfType<AnnotationModel>())
                 {
                     List<GraphLayout.Node> cluster = new List<GraphLayout.Node>();
-                    cluster.AddRange(group.SelectedModels.Select(x => combinedGraph.FindNode(x.GUID)));
+                    cluster.AddRange(group.SelectedModels.OfType<NodeModel>().Select(x => combinedGraph.FindNode(x.GUID)));
```

Comment:
So as I can understand, notes will be placed by linking with the nodes. So they are not considered here.